src/credentials/attestation_verifier/DefaultDeviceAttestationVerifier.cpp

```diff
@@ -427,10 +428,19 @@ CHIP_ERROR DefaultDACVerifier::VerifyNodeOperationalCSRInformation(const ByteSpa
 
 } // namespace
 
+#ifdef MATTER_FILE_SYSTEM_ENABLED
+const AttestationTrustStore * GetTestFileAttestationTrustStore(const char * paaTrustStorePath)
```

Comment:
Suggest to instead let the tools that need it instantiate and own this (i.e., chip-tool directly doing so)